Makefile

```diff
@@ -6,15 +6,15 @@ setup:
 
 .PHONY: compile
 compile: setup  ## Compiles the binary
-	go build -o ./bin/${SERVICE_NAME}
+	go build -race -o ./bin/${SERVICE_NAME}
```

Comment:
do you know if the warnings generated by the race detector end up in kibana?

Reply:
they are limited to build time and not runtime, so they dont end up in kibana.
Also, the race detector is exposing some mistakes in the test code